### DISCUSSION TITLE
dbt (core) and dagster in hybrid deployment

### DISCUSSION THREAD

**Discussion by my30** [dagster-io/dagster#19406] [Q&A] [answered]
Hello community, I'd like to integrate a dbt project with Dagster (in hybrid deployment with AWS ECS) and use Github Actions for the deployment workflow. 

From what I can see in [the serverless case](https://github.com/dagster-io/dagster-cloud-action/blob/main/github/serverless/dbt/branch_deployments.yml), the dbt project needs to be packed [with Dagster scaffold](https://docs.dagster.io/_apidocs/libraries/dagster-dbt#dagster-dbt-project-scaffold), built into a new container image, and gets deployed to Dagster Cloud [as a python executable](https://github.com/dagster-io/dagster-cloud-action/blob/7c2f65f979a21da1e621f768af6db5265586ee91/github/serverless/dbt/branch_deployments.yml#L61). 

But do we have any example workflow code for this use case?

**Comment by rexledesma** [comment, marked as answer]
You can check out our [Hooli repository](https://github.com/dagster-io/hooli-data-eng-pipelines), which uses Dagster Cloud Hybrid to go through some sample use cases with dbt, Databricks, Snowflake.

**Reply by my30** [reply]
Thank you, that's helpful.
